src/main/java/org/jabref/gui/externalfiletype/UnknownExternalFileType.java

```diff
@@ -11,36 +14,46 @@
  */
 public class UnknownExternalFileType implements ExternalFileType {
 
-    private final String name;
-    private final String extension;
+    private final StringProperty name;
+    private final StringProperty extension;
 
     public UnknownExternalFileType(String name) {
         this(name, "");
     }
 
     public UnknownExternalFileType(String name, String extension) {
-        this.name = name;
-        this.extension = extension;
+        this.name = new SimpleStringProperty(name);
+        this.extension = new SimpleStringProperty(extension);
     }
 
     @Override
-    public String getName() {
+    public StringProperty getName() {
         return name;
     }
 
     @Override
-    public String getExtension() {
+    public String getNameAsString() {
```

Comment:
Some code uses `.getValue()` some `...AsString()` I would always stick to `.getValue()`